### PR TITLE
refactor: update dict

### PR DESCRIPTION
**Pull request by JGSweets**
Replaces the original merge dict

### CODE REVIEW

**Review by taylorfturner — APPROVED** [review]
LGTM

**Review by micdavis — DISMISSED** [review]

**Review by taylorfturner — APPROVED** [review]

**Review by micdavis — APPROVED** [review]

File: dataprofiler/profilers/utils.py [hunk 1]
```diff
@@ -37,43 +38,22 @@
...
-    Inspired by :meth:``dict.update()``, instead of updating only
-    top-level keys, dict_merge recurses down into dicts nested
-    to an arbitrary depth, updating keys. The ``merge_dct`` is
-    merged into ``dct``.
-
-    :param dct: dict onto which the merge is executed
-    :param merge_dct: dct merged into dct
+def recursive_dict_update(d: Dict, update_d: Dict) -> Dict:
+    """
+    Recurisve updates nested dictionaries. Updating d with update_d.
```
Comment (leos) [inline dataprofiler/profilers/utils.py:43]: typo: recursive

Comment (taylorfturner) [reply dataprofiler/profilers/utils.py:43]: +1

File: dataprofiler/profilers/utils.py [hunk 2]
```diff
@@ -37,43 +38,22 @@
...
+
+    :param d: dict which gets updated with update_d
+    :param update_d: dict to update d with
     :return: None
```
Comment (taylorfturner) [inline dataprofiler/profilers/utils.py:47]: `dict`

**Pull request closed without merging**